### PR TITLE
#7 Basic test setup & #8 Postgres setup

### PR DESCRIPTION
**Pull request by archae0pteryx**
## Story

Tickets #7 & #8

## Description
- postgres
- user schema / migration
- test scripts and configs
- workflows for tests and lint
- lint fixes

## Definition of Done

- [ ] README updated
- [ ] Tests written
- [ ] Acceptance criteria implement
- [ ] Code reviewed
- [ ] Feature accepted

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -1,19 +1,12 @@
-# These environment variables will be used by default if you do not create any
-# yourself in .env. This file should be safe to check into your version control
-# system. Any custom values should go in .env and .env should *not* be checked
-# into version control.
+SESSION_SECRET='its dangerous to go alone. take this 🗡'
```
Comment (craineum) [inline .env.defaults:1]: 🔥 Don't think we should have this in here. I am good if they exists in `.env.example`, as I don't really want to fall to this default if we forget to put it in there. Would rather the app not start. Same with admin/password stuff below.

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+SESSION_SECRET='its dangerous to go alone. take this 🗡'
+
+DATABASE_URL=postgresql://postgres:development@localhost:5432/residency_development
+TEST_DATABASE_URL=postgresql://postgres:test@localhost:5433/residency_test
+
+PRISMA_HIDE_UPDATE_MESSAGE=true
+
+ADMIN_EMAIL=admin@example.com
+ADMIN_PASSWORD=password
+
```
Comment (craineum) [inline .env.example:10]: ➕ Missing `USER_PASSWORD`

**Review by craineum — COMMENTED** [review]
Couple of comments. Also make sure you update the root README with any changes/additions.

File: .env.defaults (hunk 1 above)
Comment (archae0pteryx) [reply .env.defaults:1]: 10-4

**Review by craineum — APPROVED** [review]
LGTM, squash and merge when ready 🚀

**Pull request merged**